redfish-core/lib/assembly.hpp

```diff
@@ -326,11 +331,42 @@ inline void
                                     }
                                     if (*value == false)
                                     {
+                                        // Special handling for LCD and base
```

Comment:
Nit: These lines can simply be moved to a common place instead of repeating them in the if and else clauses

Reply:
Just the comment?